### PR TITLE
tests: log upload feature

### PR DESCRIPTION
**Pull request by erikkn**
Added the feature that Terraform logs are uploaded to S3 as well.

All files are temporary stored in the 'templogfiles' directory.

### CODE REVIEW

**Comment by erikkn** [comment]
ok to test

**Comment by erikkn** [comment]
ok to test

**Comment by mxinden** [comment]
@erikkn Can you add a description to the commit so we know in the future what happened here?

File: Jenkinsfile [hunk 1]
```diff
@@ -432,6 +432,7 @@ def runRSpecTest(testFilePath, dockerArgs) {
                 sh """#!/bin/bash -ex
                   cd tests/rspec
                   bundler exec rspec ${testFilePath}
+                  cd ../..
```
Comment (mxinden) [inline Jenkinsfile]: Can you check if this is needed?

**Comment by erikkn** [comment]
ok to test

**Pull request merged**